spark/src/main/java/org/opensearch/sql/spark/dispatcher/IndexDMLHandler.java

```diff
@@ -93,22 +112,20 @@ public DispatchQueryResponse submit(
             System.currentTimeMillis());
     String resultIndex = dataSourceMetadata.getResultIndex();
     createIndexDMLResult(stateStore, resultIndex).apply(indexDMLResult);
-
+    createSessionAndStatement(
+        dispatchQueryRequest,
+        dispatchQueryRequest.getApplicationId(),
+        DROP_INDEX_JOB_ID,
```

Comment:
does that mean there is only one job id for all DML queries? Would it cause issues if there is a concurrent DML running?